yeat/cli.py

```diff
@@ -8,35 +8,43 @@
 # -------------------------------------------------------------------------------------------------
 
 import argparse
+from argparse import Action
+import json
 from pathlib import Path
 from pkg_resources import resource_filename
 from snakemake import snakemake
+import sys
 import yeat
+from yeat.config import AssemblerConfig
 
 
-ASSEMBLY_ALGORITHMS = ["spades", "megahit", "unicycler"]
+CONFIG_TEMPLATE = [
+    dict(
+        assembler="spades",
+        extra_args="",
+    ),
+    dict(
+        assembler="megahit",
+        extra_args="",
+    ),
+]
```

Comment:
Since this is for demo purposes, it would be appropriate to populate the `extra_args` argument. Maybe something like this.

```python
CONFIG_TEMPLATE = [
     dict(
         assembler="spades",
         extra_args="--meta",
     ),
     dict(
         assembler="megahit",
         extra_args="--min-count 5 --min-contig-len 300",
     ),
 ]
```